plugins/available/basher.plugin.bash

```diff
@@ -0,0 +1,12 @@
+cite about-plugin
+about-plugin 'initializes basher, the shell package manager'
+
+# https://github.com/basherpm/basher
+
+if ! _command_exists basher && [[ -x "$HOME/.basher/bin/basher" ]]; then
+	pathmunge "$HOME/.basher/bin"
+else
+	return 0
```

Comment:
maybe add a log about not being able to locate `basher`, and suggestion to install it?